src/net.cpp

```diff
@@ -129,4 +135,14 @@ std::vector<net_socket> net::get_net_sockets(const std::string& file) const
     return output;
 }
 
+std::vector<net_device> net::get_net_devices(const std::string& file) const
+{
+    auto path = _net_root + file;
+
+    static const size_t HEADER_LINES = 2;
```

Comment:
[style] Can you add a newline between this and the following one, just to keep a consistent format with other implementations (consts are separated).